### ISSUE TITLE
A11y: cannot close tab via keyboard

### ISSUE THREAD

**Issue by dpouliot** [maslianok/react-responsive-tabs#122]
The subject line I think says it all. I can navigate tabs via the keyboard but cannot close them, making them inaccessible.
🇺🇦
Fix:
`<div class="RRT__removable-icon" tabindex="0">x</div>`
